ansible/local_build.yml

```diff
@@ -0,0 +1,14 @@
+---
+
+# Clones and builds qvain-api and qvain-js locally.
+# Uses git with force so any local uncommitted
+# changes in qvain-api and qvain-js subdirs
+# are lost.
+#
+# Also creates the symlinks expected by deploy_appserver.yml
+# under qvain-ops/releases/[deployment_environment_id]
+
+- hosts: local_build
+  connection: local
```

Comment:
missed this on the first review round, just a comment, no need to change anything.

Reply:
I think I'll add `become: no` so it doesn't accidentally sudo anything.